### PR TITLE
Hide metadata from TestApp custom card rendering

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-2228

**Additional info:**
The issue does not stream from the SDK but from custom chat message rendering that has been created as an example and for testing purposes.

### CODE REVIEW

**Review by DavDo — APPROVED** [review]

File: app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java [hunk 1]
```diff
@@ -92,6 +99,16 @@ public NativeViewViewHolder(@NonNull View itemView, @NonNull UiTheme uiTheme) {
             this.okButton = itemView.findViewById(R.id.ok_button);
         }
 
+        public static boolean isNativeViewType(ChatMessage message) {
+            JSONObject metadata = message == null ? null : message.getMetadata();
+            if (metadata == null || metadata.length() == 0) {
+                return false;
+            } else {
+                return metadata.has(NativeViewViewHolder.SHOW_BUTTON_KEY)
```
Comment (AndriiHorishniiMOC) [inline app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java:107]: I am not sure that it is for these cases `NativeView` should be shown.
I'd better remove this custom view from our testing app.

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java (hunk 1 above)
Comment (gugalo) [reply app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java:107]: I think it is alright. But I can remove it as well if needed. @salemove/tm-mobile-android

Comment (AndriiHorishniiMOC) [reply app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java:107]: I mean that the `NativeView` is not necessary enough to leave it for such rather strange conditions.

**Comment by gugalo** [comment]
!merge

**Pull request merged**